db_stress_tool/db_stress_test_base.cc

```diff
@@ -613,6 +613,28 @@ void StressTest::OperateDb(ThreadState* thread) {
         }
       }
 
+      if (FLAGS_pause_background_one_in > 0 &&
+          thread->rand.OneIn(FLAGS_pause_background_one_in)) {
```

Comment:
Elsewhere in OperateDb(), we use thread->rand.Uniform(). What's the difference?

Reply:
The others are just silly for not using the simpler API